server/pulp/server/db/model/repository.py

```diff
@@ -508,6 +511,7 @@ def __init__(self, repo_id, distributor_id, distributor_type_id, started, comple
         self.repo_id = repo_id
         self.distributor_id = distributor_id
         self.distributor_type_id = distributor_type_id
+        self.distributor_config = distributor_config
```

Comment:
Please create a test that specifically asserts that this __init__() method sets this attribute.